### ISSUE TITLE
Outside-if-not working with version 2.2.1, not anymore with newer versions

### ISSUE THREAD

**Issue by rbnvrw** [IamAdamJowett/angular-click-outside#29]
When I update the plugin to a version > 2.2.1, the `outside-if-not` attribute stopped working. Small code snippet:
```html
<div  data-click-outside="deselectFormElement()" 
data-outside-if-not="editor__sidebar,option__button">
```
Downgrading to version 2.2.1 fixes the issue. Could you please fix it in a new version?

Please consider adding unit tests/e2e tests to prevent regressions. Other than that, I really appreciate your plugin!

**Comment by IamAdamJowett** [comment]
@rbnvrw I am putting tests in at the moment, but manual tests in the example app within this repo is not showing the same behavior, do you have a jsfiddle or the like for me to see the behavior?

**Comment by rbnvrw** [comment]
That is great. I do not have the time to create a jsfiddle now, but I will make one for you tomorrow.

**Comment by elliottregan** [comment]
I can second this. I haven't had time to really debug it, but I had to roll back to 2.2.1 to get it to work again.

**Comment by IamAdamJowett** [comment]
@rbnvrw and @elliottregan a quick question, are by chance either of you on a touchscreen laptop or PC? (not a tablet or mobile)

**Comment by IamAdamJowett** [comment]
@rbnvrw @elliottregan Check out release 2.5.0 and let me know if this fixes the issues you are finding.

**Comment by elliottregan** [comment]
Seems to be working again! Thanks!

I was on a regular Mac, no touchscreen.

**Comment by elliottregan** [comment]
@IamAdamJowett This bug is back. Same thing, no touch screen.

**Comment by rbnvrw** [comment]
Here also no touchscreen.

**Comment by IamAdamJowett** [comment]
@elliottregan I assume you mean 2.6.2, I will see what has changed that may have brought this back

**Comment by elliottregan** [comment]
Yup. Fixed my comment.

**Comment by IamAdamJowett** [comment]
@elliottregan fixes should be in the latest version 2.8.2 that covers this off. 5c64f22
